.gitignore

```diff
@@ -140,3 +140,8 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+
+# Docker
+database

```

Comment:
Looks great as a extra security to not pass through database while building image! 👍